crates/ra_syntax/src/grammar.ron

```diff
@@ -1,736 +1,6 @@
 // Stores definitions which must be used in multiple places
```

Comment:
Why does this file still exist?

Reply:
because I didn't review the changes :sweat_smile: